pgql-spoofax/syntax/Expressions.sdf3

```diff
@@ -47,6 +47,7 @@ context-free syntax // function calls
   Exp.AllDifferent = <all_different(<{Exp ", "}+>)> {case-insensitive}
   Exp.StX = <st_x(<Exp>)> {case-insensitive}
   Exp.StY = <st_y(<Exp>)> {case-insensitive}
+  Exp.StPointFromText = <ST_PointFromText(<STRING>)> {case-sensitive}
```

Comment:
I think here we should turn `<STRING>` into `<Exp>` to allow arbitrary expressions, for example:
`st_pointFromText(x.wktStringProp)` could be useful.

It's OK though IMO to not have it for `to_date`, `to_time` and `to_timestamp` since those are just there for legacy reasons.